src/style/color.rs

```diff
@@ -654,14 +651,20 @@ mod tests {
 
     #[cfg(feature = "serde")]
     #[test]
-    fn deserialize_error() {
+    #[should_panic = "Failed to parse Color"]
+    fn deserialize_unknown_string_errors() {
         let color: Result<_, serde::de::value::Error> =
             Color::deserialize("invalid".into_deserializer());
-        assert!(color.is_err());
+        color.unwrap();
```

Comment:
Can these be expressed as an `assert_eq!` call against `<some instance of the Error trait>.custom("failed...")` (possibly https://docs.rs/serde/1.0.202/serde/de/value/struct.Error.html ?)